README.md

```diff
@@ -3,7 +3,20 @@
 
 This repository contains example implementations of orchestrators that will consume messages from Stactize and send a response back.
 At the moment, the only orchestrator example that has been created is the Durable Function Orchestrator example. 
-  
+
+## What is an Orchestrator?
+An orchestrator is required to receive these messages, take any action required to fulfil the Event (such as updating a configuration or spinning up a new resource), and finally respond to Stactize via a service bus message posted to the Egress queue.
+
+Stactize will enqueue messages to your Ingress queue whenever a subscriber takes an action from the Azure marketplace. The message contains a JSON-formatted string that contains the information required to fulfil any action. These actions contain an Event parameter, which describes an action a user has taken:
+| Event     | Description                                               |
+| --        | --                                                        | 
+| Create    | User has signed up to your product via the marketplace    |
+| Update    | User has changed their plan or the number of seats        | 
+| Suspend   | User has failed to pay for the billing term               | 
+| Reinstate | User has resolved their billing issues                    | 
+| Delete    | User has cancelled their subscription                     | 
+
+An orchestrator can be any application that can receive and send messages via the AMQP 1.0 protocol. Connections can be achieved via [supported client libraries](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview#client-libraries) that exist for multiple languages, [direct integrations](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview#client-libraries) Microsoft and Azure-native services or the [Service Bus SDK](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-samples), which is also available for multiple platforms.  
```

Comment:
The "direct integration" link here goes to the client library anchor, and I think the sentence is missing a "with" after the the "direct integration"

Reply:
![image](https://user-images.githubusercontent.com/67424843/236228911-20fdc174-2e5f-41b8-a85b-1daf66b6b503.png)
Better?